EIPS/eip-7331.md

```diff
@@ -0,0 +1,400 @@
+---
+eip: 7331
+title: Private Equity
+description: A private equity fund Contract distributes dividends of a private equity fund in stable coins or in fund tokens as per the user's choice.
+author: Oasis Pro Markets (@tokenizeassets)
+discussions-to: https://ethereum-magicians.org/t/a-proposal-for-private-equity-and-funds-smart-contract/14681
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-07-04
+requires: 20, 3643
+---
+
+## Abstract
+
+A private equity fund is a type of investment fund that pools together capital from high-net-worth individuals, institutional investors, and sometimes, pension funds to make investments in private companies. These funds are managed by private equity firms, which specialize in acquiring ownership stakes in companies and actively managing them to generate returns.
+
+## Motivation
+
+The existing investment landscape faces multiple challenges, including high investment barriers, limited liquidity, lengthy settlement processes, regulatory compliance issues, counterparty risks, and concerns over custodial control. These barriers prevent individuals from taking control of their investments, diversifying their portfolios, and accessing the benefits of a secure and efficient investment ecosystem. There is a pressing need for a comprehensive solution that addresses these challenges, providing a tokenization platform that sidesteps investment barriers, establishes an unceasing liquid marketplace with the instant settlement, ensures regulatory compliance, eliminates counterparty risks, and enables self-custody, empowering users to make informed investment decisions, access diverse opportunities, and maintain full control over their assets.
+
+Management Fee
+When it comes to compensation, private equity funds typically charge a management fee, which is a percentage of the committed capital, to cover operational expenses. In addition, they also receive a share of the profits generated by the investments, known as carried interest. Carried interest is usually calculated as a percentage of the fund's profits above a certain threshold, often referred to as the "hurdle rate."
+
+Dividend Distribution
+During the life of the fund, the general partner may occasionally distribute interim dividends to the limited partners, who are the investors in the fund. These interim dividends are typically funded by profits generated from the portfolio companies or from the sale of certain investments. However, the primary focus remains on generating capital gains rather than distributing regular dividends. These dividend distributions can be via stable coin transfer, token minting, or via fiat currency. These distributions are handled within the smart contract itself.
```

Comment:
I highly recommend sticking with just crypto-native primitives here, not off-chain fiat. Additionally, instead of making any distinction between stable coin vs. non-stable coin, just handle ERC20s generically.

Reply:
Hi Matt, 

Appreciate the comments.  And yes, we are in agreement with the sentiment that we should be sticking to crypto-native primitives as fiat cannot be directly deposited into smart contracts.